src/network/networkd-routing-policy-rule.c

```diff
@@ -805,6 +826,12 @@ int manager_rtnl_process_rule(sd_netlink *rtnl, sd_netlink_message *message, Man
                 return 0;
         }
 
+        r = sd_rtnl_message_routing_policy_rule_get_fib_type(message, &tmp->type);
+        if (r < 0 && r != -ENODATA) {
+                log_warning_errno(r, "rtnl: could not get ip rule type, ignoring: %m");
```

Comment:
`ip rule type`-> `FIB rule type`

Could you also adjust the above `ip rule TOS` -> `FIB rule TOS`?

Reply:
```FIB rule type ```